Expensify/App
# Refactor Workspace Invite Member page

### Details


### Known issues
- Searching for a selected user shows a duplicate name. https://expensify.slack.com/archives/C01GTK53T8Q/p1633961442111100
- [x] Members are being removed from Onyx after adding. (Fixed).
- [x]  UI is breaking on Android. (Fixed)


### Fixed Issues

$ https://github.com/Expensify/App/issues/4775
$ https://github.com/Expensify/App/issues/5723

### Tests | QA Steps
1. Open NewDot.
2. Go to any workspace from the Settings page.
3. navigate to Manage members.
4. Click the Invite button.
5. Select the user from the list.
6. Click Invite members should be added to Workspace.

### Tested On


### Screenshots


#### Web | Desktop

![image](https://user-images.githubusercontent.com/24370807/137818425-89d7d8ad-1333-42ac-99d9-4db64c0b1117.png)


#### Mobile Web

![Screenshot 2021-10-19 04:50:14](https://user-images.githubusercontent.com/24370807/137819454-eb700f90-f151-474b-a32c-f3a045c466d5.png)



#### iOS


#### Android

![Screenshot 2021-10-19 04:50:14](https://user-images.githubusercontent.com/24370807/137819256-4ac27eff-4e34-4fc4-af7d-ab5cc7482de1.png)
